src/style.css

```diff
@@ -1 +1,17 @@
 /* styles go here */
+.formField {
+  display: block;
+  padding: 5px 0;
+}
+
+.fieldSet {
+  margin-bottom: 20px;
+}
+
+.lastTitle {
```

Comment:
use pseudo class last-child and first-child instead of classes lastTitle firstTitle